advanced/dapps/react-dapp-v2/src/helpers/bip122.ts

```diff
@@ -0,0 +1,72 @@
+import { schnorr } from "@noble/secp256k1";
+import * as bitcoin from "bitcoinjs-lib";
+import BitcoinMessage from "bitcoinjs-message";
+import { convertHexToBase64 } from "./utilities";
+
+export async function apiGetBip122AccountBalance(
+  address: string,
+  chainId: string
+) {
+  const utxo = await apiGetAddressUtxos(address, chainId);
+  const balanceInSatoshis = getAvailableBalanceFromUtxos(utxo);
+  const balanceInBtc = balanceInSatoshis * 0.00000001;
+  return { balance: balanceInBtc.toString(), symbol: "BTC", name: "BTC" };
+}
+
+export async function apiGetAddressUtxos(address: string, chainId: string) {
+  return await (
+    await fetch(`https://mempool.space/testnet/api/address/${address}/utxo`)
+  ).json();
+}
+
+export function getAvailableBalanceFromUtxos(utxos: any[]) {
```

Comment:
`any` seems unnecessary if it should be numbers?

```suggestion
export function getAvailableBalanceFromUtxos(utxos: number[]) {
```

Reply:
updated the type. utxos are not numbers but an object
```
 {
    "txid": "94705b1f5d1dc4b08d318f70900d4d4ef75a6dbcfde5e7e54bb3e2724320c008",
    "vout": 0,
    "status": {
      "confirmed": true,
      "block_height": 195983,
      "block_hash": "000000f85a965591ba3f25265ad43a89381b5eec8a5282dcfea4b915162eaf02",
      "block_time": 1715955664
    },
    "value": 14548
  }